### ISSUE TITLE
http usage

### ISSUE THREAD

**Issue by uniqx** [OxfordHCC/tracker-control-android#32]
Why is this not using https?

[app/build.gradle#L66](https://github.com/OxfordHCC/tracker-control-android/blob/e785360c47649ba9a54f8eb94b595a2b4347cda0/app/build.gradle)
```gradle
                buildConfigField "String", "HOSTS_FILE_URI", "\"http://www.netguard.me/hosts\""
```

**Comment by kasnder** [comment]
Good spot, and good question. It's code from NetGuard, which implements it in the same way.

Will investigate!

**Comment by kasnder** [comment]
This is fixed in commit 8321fdcfe40815f72aa95a330016ff20a59b33cf

Maybe also interesting for @M66B
